Cognigy/CustomModules
# New Custom Modules: Language Detection, TensorFlow, Sentiment Analysis

This PR introduces three new custom modules
- Language Detection module uses library for node.js to identify the language of the text.
- TensorFlow module is a template for using TensorFlow.js models in Cognigy.
- Sentiment Analysis module classifies the text into positive, negative or neutral, based on the labeled lists of words.

***How to test***
- build and install Custom Modules
- create a flow with the node you want to test
- write input text you want to test and check if the result is right in the context
